### PR TITLE
Feat login signup page

### PR DESCRIPTION
**Pull request by Vishruta-Patil**
Completed login and signup page
https://deploy-preview-5--magnizent-quiz.netlify.app/login.html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+                <span class="material-icons icon"> dark_mode </span>
+               
+              </a>
+            </div>
+          </div>
+          <span class="material-icons icon hide-menu menu-logo"> menu </span>
+        </div>
+      </div>
+  
+      <div id="empty-box"></div>
```
Comment (amarphule) [inline index.html:69]: is there any reason you kept it empty div?

File: instruction.html [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+                <span class="material-icons icon"> dark_mode </span>
+               
+              </a>
+            </div>
+          </div>
+          <span class="material-icons icon hide-menu menu-logo"> menu </span>
+        </div>
+      </div>
+  
+      <div id="empty-box"></div>
```
Comment (amarphule) [inline instruction.html:68]: same as above

File: style/quiz.css [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+.quiz-option {
+  text-align: center;
+}
+
+.option-unit {
+    background-color: var(--primary-color);
+    color: var(--secondary-color);
+    padding: 1.5rem;
+    /* max-width: 40rem; */
```
Comment (amarphule) [inline style/quiz.css:24]: remove comment

**Review by tewarimohit — APPROVED** [review]
Code looks good to merge....

**Review by amarphule — CHANGES_REQUESTED** [review]
Great work. given some comments please check

File: index.html (hunk 1 above)
Comment (Vishruta-Patil) [reply index.html:69]: That div is for overlay effect in the menu

**Pull request merged**